bindings/python/iota_sdk/types/block/block.py

```diff
@@ -64,13 +80,25 @@ class Block:
         signature: The Block signature.
     """
     header: BlockHeader
-    body: BlockBody
+    body: BlockBody = field(metadata=config(
+        decoder=deserialize_block_body
+    ))
     signature: Signature
 
     def id(self, params: ProtocolParameters) -> HexStr:
         """Returns the block ID as a hexadecimal string.
         """
         return Utils.block_id(self, params)
 
+    def to_dict(self):
+        """Custom dict conversion.
+        """
+
+        return {
+            "header": self.header.to_dict(),
+            "body": self.body.to_dict(),
+            "signature": self.signature.to_dict(),
+        }
```

Comment:
I think this shouldn't be needed, `to_dict()` gets called automatically if possible, no?

Reply:
it is needed